extensions/bittrex/exchange.js

```diff
@@ -0,0 +1,289 @@
+var bittrex_authed = require('node.bittrex.api'),
+    bittrex_public = require('node.bittrex.api'),
+    path = require('path'),
+    moment = require('moment'),
+    n = require('numbro'),
+    colors = require('colors')
+
+
+/**
+ ** Bittrex API
+ ** 
+ ** please note: unfortunately getmarkethistory does not work based on time
+ ** this means that we cannot do any backfilles, but paper trading should be fine
+ ** https://github.com/n0mad01/node.bittrex.api/issues/17
+ **
+ **/
+module.exports = function container(get, set, clear) {
+  var c = get('conf')
+  var recoverableErrors = new RegExp(/(ESOCKETTIMEDOUT|ETIMEDOUT|ECONNRESET|ECONNREFUSED|ENOTFOUND|Invalid nonce|Rate limit exceeded)/)
```

Comment:
Correct, but they do have a `nonce`. I don't think they have rate limiting yet, which is why I picked a generic message in case they add it later down the line.